tinyagentos/github_app.py

```
@@ -0,0 +1,324 @@
+) -> str | None:
+    """Return the GitHub App's slug name, or None on failure."""
+    jwt = generate_jwt(app_id, private_key)
+    try:
+        resp = await http_client.get(
+            _APP_SLUG_URL,
+            headers=_auth_headers(jwt),
+            timeout=15,
+        )
+        resp.raise_for_status()
+        return resp.json().get("slug")
+    except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)